lib/project.rb

```diff
@@ -28,6 +30,10 @@ def initialize(scope, options)
       scope << self
     end
 
+    def initialize_json_override
+      Json.update_json_override(@options.common.config) unless @options.common.config.nil?
+    end
```

Comment:
This method is too trivial. I would rather fix the Rubocop warning with `initialize` by removing `@id`.